spec/features/posts_new_message_spec.rb

```diff
@@ -0,0 +1,9 @@
+feature 'posting new messages' do
```

Comment:
perhaps could do with a few more feature tests throughout? perhaps may help for edge cases etc